KeyboardAvoidanceSwiftUI/KeyboardHeightPublisher.swift

```diff
@@ -9,16 +9,22 @@
 import Combine
 import UIKit
 
+fileprivate var keyboardUpdates: AnyCancellable?
+fileprivate let keyboardHeightPublisher: CurrentValueSubject<CGFloat, Never> = {
+    let subject = CurrentValueSubject<CGFloat, Never>(0)
+    
+    let willShow = NotificationCenter.default.publisher(for: UIApplication.keyboardWillShowNotification)
+        .map { $0.keyboardHeight }
+    let willHide = NotificationCenter.default.publisher(for: UIApplication.keyboardWillHideNotification)
+        .map { _ in CGFloat(0) }
+    keyboardUpdates = Publishers.MergeMany(willShow, willHide).assign(to: \.value, on: subject)
```

Comment:
I think it would be really slick to convert this publisher _into_ a `CurrentValueSubject` so that we didn't have to retain the return value separately, but not sure how to do that.